client/components/events/EventListView/index.ts

```diff
@@ -0,0 +1,2 @@
+export { EventListCard } from "./EventListCard"
+export { EventListView } from "./EventListView"
```

Comment:
```suggestion
export { EventListView } from "./EventListView"

```

Reply:
btw you can configure VSCode to add a newline at the end of every file when you save, if there is no newline there

Reply:
good call